digest.php

```diff
@@ -35,8 +35,8 @@
 
 // phpcs:disable Generic.Arrays.DisallowLongArraySyntax -- File needs to be parsable by PHP 5.2.4.
 
-if ( file_exists( __DIR__ . '/vendor/autoload.php' ) ) {
-	include __DIR__ . '/vendor/autoload.php';
+if ( file_exists( dirname( __FILE__ ) . '/vendor/autoload.php' ) ) {
+	include dirname( __FILE__ ) . '/vendor/autoload.php';
```

Comment:
The minimum supported version is PHP 5.6 so we do not need to support PHP 5.2.

Reply:
But the main plugin file should still be parseable by that version otherwise the requirements check doesn’t make sense.